packages/core/lib/commands/test/run.js

```diff
@@ -30,6 +32,27 @@ const parseCommandLineFlags = options => {
   }
 };
 
+// Sanitize ganache options specified in the config
+function sanitizeGanacheOptions(ganacheOptions) {
+  let network_id = ganacheOptions.network_id;
+
+  // Use default network_id if "*" is defined in config
+  if (network_id === "*") {
+    network_id = defaultNetworkIdForTestCommand;
+    return { ...ganacheOptions, network_id };
```

Comment:
This could be simplified. 

```suggestion
    return { ...ganacheOptions, network_id: defaultNetworkIdForTestCommand };
```